components/mapsV3/Map.jsx

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from 'react';
+import { ComposableMap, Geographies, Geography, Marker, ZoomableGroup } from 'react-simple-maps';
+import { classes } from '../../utils';
+import { Button } from '../buttons';
+import { MarkerIcon } from '../icons';
+import { Skeleton } from '../skeleton';
+import { Tooltip } from '../tooltip';
+import styles from './Map.module.css';
+import countries from './countries.json';
+
+const Map = (props) => {
+	const {
+		loading,
```

Comment:
- [ ] remove loading, theme, fallback
- [ ] initiate className and coordinated with default values